__tests__/app.test.js

```diff
@@ -433,4 +433,7 @@ describe("DELETE /api/comments/:comment_id", () => {
   test("status:204, responds with an empty response body", () => {
     return request(app).delete("/api/comments/5").expect(204);
   });
+  test("status:400, responds with bad request when invalid input", () => {
```

Comment:
What if the user tries to delete a commentId that IS a number, but doesn't exist?